.werf/defines/image-labels.tmpl

```diff
@@ -0,0 +1,16 @@
+# oci image labels with supported k8s versions
+# applies on the main deckhouse image and on the install image
```

Comment:
Can you add parameters descriptions contain in the context?
As here, for example https://github.com/deckhouse/deckhouse/blob/b98edcdbc35b4f63cdff6ab664933a0330f14026/.werf/defines/dhctl.tmpl#L1-L3

Reply:
yes

Reply:
done